Oracle/PCORNetLoader_ora.sql

```diff
@@ -1009,12 +1009,48 @@ execute immediate 'create index pdxfact_idx on pdxfact (patient_num, encounter_n
 GATHER_TABLE_STATS('PDXFACT');
 
 insert into diagnosis (patid,			encounterid,	enc_type, admit_date, providerid, dx, dx_type, dx_source, pdx)
-select distinct factline.patient_num, factline.encounter_num encounterid,	enc_type, enc.admit_date, factline.provider_id, diag.pcori_basecode,
-SUBSTR(diag.c_fullname,18,2) dxtype,
-	CASE WHEN enc_type='AV' THEN 'FI' ELSE nvl(SUBSTR(dxsource,INSTR(dxsource,':')+1,2) ,'NI') END,
+/* KUMC started billing with ICD10 on Oct 1, 2015. */
+with icd10_transition as (
+  select date '2015-10-01' as cutoff from dual
+)
+/* DX_IDs may have mappings to both ICD9 and ICD10 */
+, has9 as (
+  select distinct c_basecode, pcori_basecode icd9_code
+  from blueheronmetadata.pcornet_diag diag

```

Comment:
We use replacement variables in other places rather than KUMC-specific schemas.  Perhaps worth changing here.